pkg/odo/cli/component/link.go

```diff
@@ -24,12 +24,6 @@ var (
 	linkExample = ktemplates.Examples(`# Link the current component to the 'EtcdCluster' named 'myetcd'
 %[1]s EtcdCluster/myetcd
 
-# Link the current component to the 'my-postgresql' service
-%[1]s my-postgresql
-
-# Link component 'nodejs' to the 'my-postgresql' service
-%[1]s my-postgresql --component nodejs
-
```

Comment:
This should not be deleted, this is still relevant for Operator backed services.

Reply:
change reset